src/lib/registration/connect_helpers.rb

```diff
@@ -157,6 +168,15 @@ def self.report_error(msg, error_message)
       Yast::Report.Error(error_with_details(msg, error_message))
     end
 
+    def self.retry_error(msg, error_message)
+      buttons = { retry: _("Retry"), cancel: _("Cancel") }
+      Yast2::Popup.show(msg, details: error_message, headline: :error, buttons: buttons)
```

Comment:
We really need to use `Yast::Report` module here, the `Yast2::Popup` module does not support AutoYaST mode.

Or we should handle it here, e.g. in AY mode use `Yast::Report` (without retrying, if that's not possible) and in the interactive mode use `Yast2::Popup`.

Reply:
Ideal solution for me is to have `Yast2::Report` but for that I need a bit time. Basic idea is to have same syntax as Yast2::Popup, just add param with severity and also setup Report configuration for that new Report. So not hard, but need time to implement.